alternator/executor.cc

```diff
@@ -1027,7 +1027,7 @@ static lw_shared_ptr<query::read_command> read_nothing_read_command(schema_ptr s
 }
 
```

Comment:
changelog should explain why this is a good thing.

Reply:
I added a commit message.